happi/cli.py

```diff
@@ -396,23 +396,48 @@ def edit(ctx, name: str, edits: list[str]):
 
 
 @happi_cli.command()
-@click.argument('item_names', nargs=-1)
+@click.option('--glob/--regex', 'use_glob', default=True,
+              help='use glob style (default) or regex style search terms. '
+              r'Regex requires backslashes to be escaped (eg. at\\d.\\d)')
+@click.argument('search_criteria', nargs=-1)
 @click.pass_context
-def load(ctx, item_names: list[str]):
+def load(
+    ctx: click.Context,
+    use_glob: bool,
+    search_criteria: list[str]
+):
+    """ Checks if the search string contains a glob pattern. If so,
+    it's sent to the search_parser() function. """
+
+    item_names = []
+
+    if '*' not in (' '.join(search_criteria)):
+        item_names = search_criteria
+    else:
+        final_results = search_parser(
+            client=get_happi_client_from_config(ctx.obj),
+            use_glob=use_glob,
+            search_criteria=search_criteria,
+        )
+        if not final_results:
+            return []
+
+        for res in final_results:
+            item_names.append(res['name'])
+
     """Open IPython terminal with ITEM_NAMES loaded."""
```

Comment:
For pep8-ness, this should be a comment with the  normal `#` character.  the triple quotes are usually reserved for docstrings or block strings